src/main/java/github/weichware10/util/data/DataPoint.java

```diff
@@ -15,43 +17,47 @@
     public final Float zoomLevel;
 
     /**
-     * Stores a single DataPoint with zoomLevel.
+     * Konstruktor für Jackson.
```

Comment:
Heißt das wirklich Jackson ?

Reply:
Jackson ist der Name des Moduls, welches Instanzen von Klassen in .json umwandelt bzw. .json in Instanzen umwandelt.